### PR TITLE
Add proto_crypt_secret_free

### PR DESCRIPTION
**Pull request by gperciva**

### CODE REVIEW

File: proto/proto_crypt.c [hunk 1]
```diff
@@ -109,15 +110,19 @@ proto_crypt_secret(const char * filename)
...
 
 err2:
+	/* Close the file if it isn't stdin. */
 	if (f != stdin)
 		fclose(f);
+
+	/* Wipe context state. */
+	SHA256_Final(K->K, &ctx);
 err1:
 	free(K);
```
Comment (cperciva) [inline proto/proto_crypt.c:127]: Should this be `proto_crypto_secret_free(K)` here?  Because we've just written keying material into `K->K`.

Comment (gperciva) [reply proto/proto_crypt.c]: Yes, added as a REBASE commit.

**Comment by cperciva** [comment]
Please rebase.

**Comment by gperciva** [comment]
Done

**Pull request merged**